baseapp/baseapp.go

```diff
@@ -106,6 +106,9 @@ type BaseApp struct { // nolint: maligned
 
 	// application's version string
 	appVersion string
+
+	// debug flag turns on more error reporting
+	debug bool
```

Comment:
Can we call this trace or something?

Reply:
I will rename while I fix the linter errors